tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py37,py38,py39,py310,py311,py312,py313
+envlist = py38,py39,py310,py311,py312,py313
```

Comment:
Also needs to be updated in `README.md`, `setup.py`, and `.github/workflows/test.yml`